modules/ding_page/ding_page.pages_default.inc

```diff
@@ -329,5 +329,370 @@ function ding_page_default_page_manager_handlers() {
   $handler->conf['display'] = $display;
   $export['node_view_panel_context_4'] = $handler;
 
-  return $export;
+    $handler = new stdClass();
```

Comment:
@tripox This should probably not be here. Could I just move this to the bbs_pages module?

Reply:
Correct. Yes, @reon-g. :)